vaadin-upload-flow-parent/vaadin-upload-flow/src/test/java/com/vaadin/flow/component/upload/tests/UploadTest.java

```diff
@@ -0,0 +1,13 @@
+package com.vaadin.flow.component.upload.tests;
+
+import com.vaadin.flow.component.upload.Upload;
+
+import org.junit.Test;
+
+public class UploadTest {
+
+    @Test
+    public void uploadNewUpload() {
```

Comment:
@TatuLund Could you please add a comment that explains the purpose of this test to make it clear?